ui/keys/prKeys.go

```diff
@@ -24,6 +25,10 @@ type PRKeyMap struct {
 }
 
 var PRKeys = PRKeyMap{
+	Approve: key.NewBinding(
+		key.WithKeys("!"),
```

Comment:
I think <kbd>v</kbd> would work better here? as in a checkmark?
maybe we can change to <kbd>a</kbd> when we introduce a breaking change.

Reply:
That's great, changed!